tests/common/cached_transformers_test.py

```diff
@@ -14,6 +19,44 @@ def test_get_missing_from_cache_local_files_only(self):
                 local_files_only=True,
             )
 
+    def test_from_pretrained_avoids_weights_download_if_override_weights(self):
+        # only download config because downloading pretrained weights in addition takes too long
```

Comment:
I think this comment needs updating, right? We do download the weights here?

Reply:
We do not, we only download the config here because downloading the weights would take too long with 'bert-base-uncased', which would stall the tests. But, this might be possible with epwalsh/bert-xsmall-dummy.

Reply:
Ohh, I see. Well no need to download the weights for `bert-xsmall-dummy` either then. But it would still speed up the test to use that instead of the real BERT model.